cli/lib/kontena/cli/master/user/invite_command.rb

```diff
@@ -39,11 +39,11 @@ def execute
             puts "  * command: kontena master join #{current_master.url} #{response['invite_code']}"
           end
           roles.each do |role|
-            Kontena.run(["master", "user", "role", "add", role, email])
+            raise "Failed to add role" unless Kontena.run?(["master", "user", "role", "add", role, email])
```

Comment:
I wonder if this kind of thing would be easier if `Kontena.run` just didn't propagate `SystemExit`, re-raising it as something that's a `StandardError`?